common/metrics.md

```diff
@@ -6,24 +6,24 @@ To provide full [3D Observability](https://www.wavefront.com/wavefront-enhances-
 * Errors – the number of failed requests per second.
 * Duration – per-minute histogram distributions of the amount of time that each request takes.
 
-The `WavefrontTracer` derives the RED metrics from your spans automatically, with no additional configuration or instrumentation on your part. To visualize these out-of-the-box metrics and histograms in Wavefront:
+The `WavefrontTracer` derives the RED metrics from your spans automatically, with no additional configuration or instrumentation on your part. To visualize the out-of-the-box metrics and histograms in the default Wavefront Service Dashboard:
 
-1. Select **Applications -> Inventory** in the Wavefront task bar to display the Application Services page. If necessary, scroll to find your application and its services.
-2. Click on the service you want to see metrics for. If necessary, click on **Overall** to expand the section.
+1. Select **Applications -> Service Dashboard** in the Wavefront taskbar. 
+1. Use the filters to find your application and its services.
 
-The **Overall** section displays the derived RED metrics for the service in these auto-generated charts: Request Rate, Error Rate, Duration (P95), Top Requests, Top Failed Requests, and Slowest Requests.
+The Service Dashboard displays the derived RED metrics for a service in these auto-generated charts: Request Rate, Error Rate, Duration (P95), Top Requests, Top Failed Requests, and Slowest Requests.
 
 The following table shows the name and type of the RED metrics that the Wavefront Tracer derives from your spans:
```

Comment:
Do I know what "the Wavefront Tracer" is?

Reply:
Users are directed to this page for more information.
For example: From the Opentarcing Java SDK we send the users here.. By the time the users come to this page, they have an idea about the tracer. Therefore, I don't think we should link to another page as it will confuse a user (they come from different SDKs to this page, which is a common page).

What do you think?